src/redux/flows/flows.reducer.ts

```diff
@@ -159,6 +164,10 @@ function _deleteFlow(state: FlowState, flowId: string): FlowState {
   return stateWithoutFlow;
 }
 
+function _importFlow(state: FlowState, customYAML: string): FlowState {
```

Comment:
BTW, in the future we will support files so this function will be expanded. For now it just pipes to `_createNewFlow`